### ISSUE TITLE
Banner - variant should be indicated by more than just color

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-design#1159]
cc @mcarrano 

Context for this issue: https://github.com/patternfly/patternfly-react/issues/5890#issuecomment-1122357757

The issue linked above aims to resolve a lack of context of banner variant/severity for screen reader users. As noted in the linked comment above, users who do not rely on a screen reader but have a color deficiency may have trouble understanding the variant/severity of a banner since only color is used for visual context.

The proposed change was to include icons in banners (could either be only some or all variants) to offer more context (somewhat similar to how the [alert component](https://www.patternfly.org/v4/components/alert) is designed).

**Comment by yichen1yu** [comment]
[Here](https://marvelapp.com/prototype/17i95gf4/screens) is the banner with icons, along with two use case samples. Icons are the same set that used for alert to keep consistency. https://marvelapp.com/prototype/17i95gf4/screens

**Comment by yichen1yu** [comment]
Sketch editable link: https://www.sketch.com/s/573c5ca7-e1b9-4b65-adfa-4a7834c6de08

**Comment by mcarrano** [comment]
@mmenestr @doruskova Do we need follow up doc and design kit issues for this? I think so. If you agree, please add when you get a chance. Thanks.

**Comment by mmenestr** [comment]
I think it should be added yes - [created this issue](https://github.com/patternfly/patternfly-org/issues/3117) @mcarrano

**Comment by doruskova** [comment]
Yes! @mcarrano I created an issue for it https://github.com/patternfly/patternfly-design-kit/issues/564
